### ISSUE TITLE
 Melee reach attack  way too effective  and can be easily exploited

### ISSUE THREAD

**Issue by Firestorm01X2** [CleverRaven/Cataclysm-DDA#23790]
Current melee reach attack can be easily exploited.

Game version:
0.C.27487

Operating system:
Win X64

Tiles or curses:
Any

Mods active:
Deafault

Zero skill chracter engages group (10-15) of "normal speed" zombies using simple spear.

### Expected behavior
Player takes out some of zombies but eventially start getting damage.

### Actual behavior
Player easily kites zombies  and kills it without taking damage using combination of running and reach attacks.

### Steps to reproduce the behavior
Get any  simple spear and try to engages group of zombies with zero skill character.

### Probable solutions:
* Increase stamina loss on reach attack.
* Increase time to complete reach attack. **It is already ingame**
* Add risk of losing wepon during reach attack. 
* Add effect of "losing balance" after missing reach attack that increase time to complete attack and make character vulnurable to enemy. **It is already ingame.**
* Make starting spear extremly fragile.

**Comment by smolbird** [comment]
That's arguably the point. You will get swarmed unless you take measures to avoid that. This applies to all combat in general.

**Comment by BorkBorkGoesTheCode** [comment]
A different way to nerf this would be to penalize maneuvering with long weapons inside narrow cluttered spaces and/or add a risk of disarming if many enemy close within the range of the reach weapon while on open ground.

**Comment by Firestorm01X2** [comment]
Yes, but reach are attacks are much safer. Alternative may be reducing resource of beginning hand made spear to makebit fall apart after extensive use.

**Comment by Theundyingcode** [comment]
> beginning hand made spear to make it fall apart after extensive use quicker.

It would make sense that a knife spear would need more maintenance than a cruddle given it has twice the parts.

**Comment by Firestorm01X2** [comment]
Also how about going risk/reward thing. What if we add effect of losing balance for missing reach attack. Thisvway spers will work as usual until you actually can hit things, but if you miss then you lose more time and syffer consiquences.

**Comment by YakumoChen** [comment]
have you ever actually tried to attack with a spear as opposed to swinging a sword? Spears are extremely easy to attack with, you just have to thrust forwards. Little movement is needed to attack.

Compare to swinging any object, be it a sword, a club, etc. Get off your chair and try to find a method of swinging around a huge chunk of metal that's more efficient then a thrust of a spear. There's a reason spears are a mainstay of war, even to this day with bayonets. The skill floor for learning to attack with a spear is much, much lower as opposed to a sword or many other weapons of war.

The game is all about kiting in general anyway, use bushes to swipe and kite, use spears to run a step back, swing, and kite, it's all the same. Spears just put you an extra space away from the zombies, and that's completely the point.

**Comment by W5Rusk** [comment]
yes, but realism should be encouraged within the balance of game mechanics. When realism breaks the balance, its time comes out, and the time of the game convention comes. So the realism is that killing someone with a spear is a laborious process anyway, and you will be exhausted much faster than you kill a large number of enemies. Do not believe me? Go to the ring and fight. You will be unpleasantly surprised how quickly you are exhausted even without a spear, which burdens your hands. Also, the spear that destroys the hordes is a piece of scrap, wrapped in cowards with a stick, you can be hacked to kill them and one zombie before it collapses. This is by the way about realism.

**Comment by 1skandar** [comment]
Right, I'm going to call bullshit on the original post. Take a zero skill character, make a spear, run into the middle of a typical town...and you will die. Or be forced to retreat without killing many zombies at all. Given the *at least* three to four swings per zombie you'll need, you'd be forced back by about 40 or so tiles while running the entire time. That...is just not going to happen without something going wrong and you getting swarmed under most of the time.

Now, if you want to single out one or two zombies at a time and kite them to death that way....yeah, that you can do. And that is what a spear is good for. And a proper cautious player can take out a lot of zombies that way with little risk. But I wouldn't necessarily call that a problem, really.

**Comment by Firestorm01X2** [comment]
>Right, I'm going to call bullshit on the original post. Take a zero skill character, make a spear, run into the middle of a typical town...and you will die. Or be forced to retreat without killing many zombies at all. 

I mentioned kiting in the first place.  Yes in this case you will die. But if you have place to treat - then you will kite pretty efficently and esily- without using bushes or other terrain features.

**Comment by 1skandar** [comment]
2 or at most 3 zombies at a time...maybe if things go perfectly. 10 will get a zero skill character killed every time. Youll get backed up against terrain, or forced into amother group of zombies, or misjudge the timing and get grabbed or one of the many other things that can go wrong. I think you overestimate how quickly a zero skill character can kill zombies. Youll need two or three hits per and youll miss at least once and more often two or three times for each hit you land. And you’ll need to run away two tiles per swing, so we are talking a running battle across 50 or 60 tiles of space and, realistically, youre not going to get it. 

Now, if you want to say that a spear allows you to tackle one to three zombies without much, if any, risk ill agree with you. I will also say “So what?” There enough zombies out there that will ruin this tactic where it really wont help that much beyond the very early game.

**Comment by DoctorGoat** [comment]
Don't you already take more time to do a reach attack and stumble if you miss?

**Comment by FulcrumA** [comment]
I wouldn't like artifically changing stats of reach wepaons, making them weaker, require mor repair etc. There are more realistic ways of balancing such. For example, many reach weapons become cumbersome when there's less space to use them - thus making them harder to use offensively when the enemy is just next to the character. 

Mimicking it (just small, but noticeable increase in movement required and higher chance to miss unless character has so much strength they can just sweep the enemy) would be more than enough balance especially since - with the exception of awl pike - stats on majority of reach weapons aren't anything to write home about.

**Comment by papersplease** [comment]
In fact, this issue has been created after a somewhat heated discussion on another forum. I can vouch for OP claim after one of the participants (@W5Rusk ) streamed it live (zero skill character, dozens of zombies taken out with zero received damage). Stamina is another issue actually, with 0 mouth encumbrance the regeneration is just too fast, allowing to fight hordes with an unskilled character.

**Comment by Firestorm01X2** [comment]
Here is that stream record:
https://www.twitch.tv/videos/263920603

Watch out! Somewtime bad Russian language involved.

**Comment by Firestorm01X2** [comment]
Too sad there is no full record.

**Comment by papersplease** [comment]
@Firestorm01X2 I don't think it was archived properly, unfortunately.

**Comment by Firestorm01X2** [comment]
>In fact, this issue has been created after a somewhat heated discussion on another forum. I can vouch for OP claim after one of the participants (@W5Rusk ) streamed it live (zero skill character, dozens of zombies taken out with zero received damage).

I can prove that. I've seen it. And in fact I was surpised. I think he will die after encountering fast zombies. But he managed to deal with them.

**Comment by smolbird** [comment]
> I can vouch for OP claim after one of the participants 

What types of zombies? And what world settings? Were zombie speeds at 100%?

**Comment by papersplease** [comment]
@smolbird Default world settings (no hordes), Evacuee/Survivor 8-8-8-8 with no skills or traits. He was really careful and managed to clear a region in a city big enough to not be attributed to luck, without exploiting obstacles, just with accurate kiting of several zombies at once and watching the aggro indicator. The method demonstrated should work with any melee enemy, even hulks. I have no reason to think it wouldn't work with hordes either, it will just take more time. He was lucky to not encounter any shockers though, but they could have been dispatched with some obvious HP/pain tax.

**Comment by papersplease** [comment]
he's streaming a similar challenge right now btw https://www.twitch.tv/wrusk , but with a 4-4-4-4 character.

**Comment by smolbird** [comment]
I still find this extremely questionable, as shockers, feral runners, feral predators, spitters, smokers, bloated zombies, necromancers, and masters will all derail this entire kiting tactic.

There are enough things that can mess with a player that tries this that I am still of the opinion that you want to punish players for using their brains.

**Comment by Zireael07** [comment]
@smolbird: If this was the beginning of the game and Hordes off, it's unlikely that the player will run into any of the mentioned...

**Comment by CoroNaut** [comment]
I always see all of the above within the first 5-10 minutes of starting a new game, might just be my difficulty settings though. I have to say that the issue isn't really an issue, easy fix would be to increase difficulty because it seems like OP has it too easy :D

**Comment by 1skandar** [comment]
Feral runners, spitters, smokers (especially), bloated zombies, and boomers are all decently common from the very start and all will give a kiting player a headache.

Also, it should be noted that even a knife spear takes survival 1 and fabrication 1 plus raw materials to make.While not difficult to do, by any means, it is still a decent investment in time and materials for a new player who starts with not very much..

And it isn't like it is the only way to make early combat safe(er). Throwing sticks and even bows and slings can *also* be gotten rather easily. With the right skills, so can a crossbow. 

I'm seeing no real good argument here why this is a bad thing, instead this is looking more like difficulty just for difficulty's sake" and I dare say this can be closed.

**Comment by smolbird** [comment]
Agreed, this issue is invalid.

**Comment by Night-Pryanik** [comment]
> Throwing sticks and even bows and slings can also be gotten rather easily. With the right skills, so can a crossbow.

All of this require constant crafting (and/or reacquiring of arrow/bolts) of ammo, while knife spear is a craft-once-and-forget weapon.

**Comment by Firestorm01X2** [comment]
Perhaps compromise solution can be that: make easy to craft spear pretty fragile.

**Comment by 1skandar** [comment]
I still call this a non-issue. While spears may currently be the most visible thing right now, I can easily point out quite a few other way to achieve the same effect, and I'm sure there are many other methods I haven't thought about. 

Singling out a few zombies under correct conditions and killing them without allowing them into melee range is just smart gameplay and the game provides lots of tools to do so. Players availing themselves of those methods isn't an issue and isn't something that needs changing. Even beyond that, any change you make here is not going to impact experienced players one bit, but it will make the already difficult learning curve for inexperienced characters just that much worse and I fail to see how that adds anything positive to the game.

**Comment by BorkBorkGoesTheCode** [comment]
That would apply to other repetitive actions.

**Comment by AMurkin** [comment]
Depends on the implementation. Weight and damage of melee weapon versus stats of PC.

**Comment by DoctorGoat** [comment]
https://github.com/CleverRaven/Cataclysm-DDA/issues/23790#issuecomment-390662695

**Comment by kevingranade** [comment]
Expected result is invalid, vanilla zombies are expected to be kiteable with entry-level weapons.

**Comment by Firestorm01X2** [comment]
@kevingranade
But are spears supposed to be most effective beginning weapon?
